src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -269,6 +270,21 @@ private function collectPaths(ApiResource $resource, ResourceMetadataCollection
             $openapiOperation = $openapiOperation->withParameters($openapiParameters);
             $existingResponses = $openapiOperation->getResponses() ?: [];
             $overrideResponses = $operation->getExtraProperties()[self::OVERRIDE_OPENAPI_RESPONSES] ?? $this->openApiOptions->getOverrideResponses();
+            if ($operation instanceof HttpOperation && null !== $operation->getErrors()) {
+                foreach ($operation->getErrors() as $error) {
+                    /** @var ProblemExceptionInterface $exception */
+                    $exception = new $error();
+
+                    $operationErrorSchemas = [];
+                    foreach ($responseMimeTypes as $operationFormat) {
+                        $operationErrorSchema = $this->jsonSchemaFactory->buildSchema($error, $operationFormat, Schema::TYPE_OUTPUT, null, $schema, null, $forceSchemaCollection);
+                        $operationErrorSchemas[$operationFormat] = $operationErrorSchema;
+                        $this->appendSchemaDefinitions($schemas, $operationErrorSchema->getDefinitions());
+                    }
+
+                    $openapiOperation = $this->buildOpenApiResponse($existingResponses, $exception->getStatus(), $exception->getType(), $openapiOperation, $operation, $responseMimeTypes, $operationErrorSchemas, $resourceMetadataCollection);
```

Comment:
`type` on problem details is:

> The "type" member is a JSON string containing a URI reference [[URI](https://datatracker.ietf.org/doc/html/rfc3986)] that identifies the problem type. Consumers MUST use the "type" URI (after resolution, if necessary) as the problem type's primary identifier.When this member is not present, its value is assumed to be "about:blank".

https://datatracker.ietf.org/doc/html/rfc9457#name-type

we should probably use something else like `$operation->getDescription()` or `$exception->getDetails()`.